src/frontend/src/pages/Dashboard/Dashboard.jsx

```diff
@@ -3,12 +3,25 @@ import { Button, Grid, Column } from '@carbon/react';
 import { useNavigate } from 'react-router-dom';
 import AppRoutes from 'navigation/AppRoutes';
 import logo from 'assets/logo.jpg';
+import useAuth from 'hooks/AuthHooks';
 
-function Dashboard() {
+function Banner() {
+    const { session, logout } = useAuth();
     const navigate = useNavigate();
 
-    return (
-        <Grid>
+    if (session?.user != undefined) {
+        console.log(session.access_token)
```

Comment:
for now, let's keep this `console.log` but within the auth hook for ease-of traceability. This is the kind of "testing" code that MVPs often accidentally copy all over the project